src/app/components/WinModal.ts

```diff
@@ -1,38 +1,53 @@
 import { Player } from '../../Player';
 import { Render } from '../utils/Render';
 import { ViewController } from '../ViewController';
-import { BasicModal } from './BasicModal';
+import { EmptyModal } from './EmptyModal';
 
-export class WinModal extends BasicModal {
+export class WinModal extends EmptyModal {
   constructor(private view: ViewController) {
     super();
   }
 
   createWinModal(player: Player): HTMLElement {
-    this.renderBasicModal(
-      'CONGRATULATIONS!',
-      `${player.theName} has won!`,
-      this.createImage(player),
+    const heading = Render.elementFactory(
+      'h2',
+      { className: 'modal__win-heading' },
```

Comment:
Perhaps we could have a modal__heading class which would be the same for all the modals?

Reply:
done